API/Controllers/WeatherForecastController.cs

```diff
@@ -12,22 +15,18 @@ public class WeatherForecastController : ControllerBase
     };
 
         private readonly ILogger<WeatherForecastController> _logger;
+        private readonly SightseeingdbContext _context;
```

Comment:
it is awfull practice to keep context in controller. I hope it is only for test purposes